hoot-services/pom.xml

```diff
@@ -136,7 +136,7 @@
                     parallelism, but haven't been able to get them to work...so forking instead. -->
                     <!-- parallel>methods</parallel>
                     <useUnlimitedThreads>true</useUnlimitedThreads -->
-                    <forkCount>4</forkCount>
+                    <forkCount>16</forkCount>
```

Comment:
Oh, ok i see.  forkCount is the only other thing coming to mind that was being replaced there besides the db info...  was originally part of #372.  If that change can get back in there, that'd be great.